### PR TITLE
created share buttons using share urls

### PR DESCRIPTION
**Pull request by weslers**
Instead of using the standard tweet, like button etc which can have performance issues I created simple links to match the header are using the relevant share links which we can add in the title, link, description etc. There is no social media icons in materialize set so for now just used the reply icon. 

Thoughts on this layout or improvements

### CODE REVIEW

File: client/templates/initiative/header/initiativeheader.html [hunk 1]
```diff
@@ -19,6 +19,16 @@ <h2 class="center-align valign">{{title}}</h2>
         </div>
         {{/unless}}
         {{/if}}
+        <div class="social-share-buttons">
+          
+          <a target="_blank" class="twitter-share-button" href="https://twitter.com/share?url={{getCurrentUrl}}&text={{title}}"><i class="material-icons tiny">reply</i> Twitter</a>
```
Comment (trustieee) [inline client/templates/initiative/header/initiativeheader.html]: Should encode ampersands even in attributes. `&amp;`

**Comment by weslers** [comment]
Fixed that now

**Comment by trustieee** [comment]
Here's an example of what it looks like for reference to others.

example

**Comment by iamlasse** [comment]
Looks good, only thing, and this also relates to the votes and comments, might not be obvious they are clickable. Been thinking about if the votes and comments in header should just be indication or actual actions. But good start Wes!

**Comment by weslers** [comment]
Updated version. What do you guys think

screen shot 2015-07-03 at 21 00 04

**Comment by iamlasse** [comment]
Very cool! I dig it. We can always replace icons when marios icon package is done.

**Comment by weslers** [comment]
Tried to keep in with the style that was already there with the notifications and follow icons maybe not having any icon beside the name might work?

**Comment by weslers** [comment]
We could always use the wave button style in materialize???

**Comment by jeffshaver** [comment]
I don't think i particularly like the buttons. I liked them more without the borders lol. They look more like buttons with the border but at the same time, I think they take look weird on something that otherwise is minimal

**Comment by trustieee** [comment]
My preference is to have icons instead of text. I'm also not sure if we should put them in the header image.
- Header images are going to be user-driven eventually, so we can't be sure text or icons will blend nicely with the background
- I think we should use a single color scheme for icons instead of the colors that typically represent the social media mediums

**Comment by weslers** [comment]
What about something like this http://tympanus.net/Development/CreativeGooeyEffects/share.html

**Comment by trustieee** [comment]
Those look cool, but there may be other options that cater the Cherish design.

Maybe something like this:
- https://www.iconfinder.com/icons/325018/facebook_logo_social_network_square_web_icon#size=128
- https://www.iconfinder.com/icons/279409/facebook_fb_logo_media_social_icon#size=128

**Comment by weslers** [comment]
If you want to let me know what way you want it to look I can update it.

**Comment by trustieee** [comment]
Closing this while we figure out which way to go with it. Can be re-opened later.

**Comment by iamlasse** [comment]
I like the round buttons. Simple and clean and in line with the current
design. I agree icons are preferred over text and placement in header can
be tricky since we cant rely on the colors etc having the default look and
feel.

**Pull request closed without merging**